packages/ui/src/bounty/modals/WithdrawSignModal/WithdrawSignModal.tsx

```diff
@@ -47,7 +47,11 @@ export const WithdrawSignModal = ({
     transaction,
     signer: controllerAccount.address,
   })
-
+  /*
+   *   FIXME: as bounty creator your cherry is transformed as contibution, so with bounty fails with no contributions
+   *    member is told that he can withdraw cherry multiplied by 2. Cherry / number of contributors, which is 123,
+   *     and cherry as contribution.
+   * */
   const extraAmount = bounty && isContributor ? bounty.cherry.toNumber() / (bounty.contributors.length || 1) : 0
```

Comment:
The cherry should be split [pro-rata](https://joystream.gitbook.io/testnet-workspace/system/bounties#assurance-contracts-and-dominant-assurance-contracts) of the contributions so the calculation should be something like `cherry * amount / totalFunding` (see [`get_cherry_fraction_for_member`](https://github.com/Joystream/joystream/blob/6cdd8e47294bdca09f0f5d0bc04de75dea4f72e3/runtime-modules/bounty/src/lib.rs#L1570)) but with BN (JOY amounts might get greater than the `MAX_SAFE_INTEGER`).
Also the cherry doesn't get transformed into a contribution so there's no need for the FIXME